doc/i2p.md

```diff
@@ -0,0 +1,72 @@
+# I2P support in Bitcoin Core
+
+It is possible to run Bitcoin Core as an
```

Comment:
Not sure if I should drop 22250 or dashify, I don't THINK we support i2p at this point, do we?

Reply:
we don't, should drop 22250 for now